examples/seismic/acoustic/wavesolver.py

```diff
@@ -18,9 +18,9 @@ class AcousticWaveSolver(object):
     ----------
     model : Model
         Physical model with domain parameters.
-    source : PointData
+    source : AcquisitionGeometry
         The injected wave.
-    receiver : PointData
+    receiver : AcquisitionGeometry
```

Comment:
Euh no, remove both src/rec (see function signature only Geometry as input)

Geometry: AcquisitionGeometry

same everywhere

Reply:
So you want like : 
    geometry : AcquisitionGeometry
        Geometry object that contains the source (SpareTimeFunction) and 
        receivers (SparseTimeFunction) and their position.

Reply:
yes

Reply:
ok